risc0/circuit/rv32im-sys/cxx/extern.cpp

```diff
@@ -334,4 +347,39 @@ extern_plonkReadAccum_bytes(void* ctx, size_t cycle, const char* extra, std::arr
   return {item.elems[0], item.elems[1], item.elems[2], item.elems[3]};
 }
 
+void extern_syscallBigInt2Precompute(void* ctx,
+                                     size_t cycle,
+                                     const char* extra,
+                                     std::array<Fp, 0> args) {
+  // no-op
+}
+
+std::array<Fp, 16>
+extern_syscallBigInt2Witness(void* ctx, size_t cycle, const char* extra, std::array<Fp, 5> args) {
+  PreflightTrace* trace = static_cast<MachineContext*>(ctx)->trace;
+  size_t extraIdx = trace->cycles[cycle].extraIdx;
+  uint32_t a0 = trace->extras[extraIdx + 0];
+  uint32_t a1 = trace->extras[extraIdx + 1];
+  uint32_t a2 = trace->extras[extraIdx + 2];
+  uint32_t a3 = trace->extras[extraIdx + 3];
+  return {
```

Comment:
Nit: you do this as a look for the cuda version, why not here?

Reply:
Ah, I guess I could do a loop here too.